raft/src/main/java/io/zeebe/raft/RaftMembers.java

```diff
@@ -100,7 +100,7 @@ public RaftMember removeMember(final int nodeId) {
     if (member != null) {
       members.remove(member);
       memberLookup.remove(nodeId, member);
-      persistentStorage.removeMember(nodeId);
+      persistentStorage.removeMember(nodeId).save();
```

Comment:
We can then remove the save in Raft#memberLeaves right?

Reply:
Yes, sorry wasn't aware that we called save there.